### ISSUE TITLE
[Feature Request] Easier Window Dragging Between Workspaces

### ISSUE THREAD

**Issue by feren** [linuxmint/Cinnamon#5803]
It would be nice to be able to have the workspaces switch with the window after a small delay of the mouse dragging the window being on the exact side of the screen or even on mouse pressure do the same thing. There's something similar on Compiz, etc where the workspace the mouse is pushing towards peeks onto the screen slightly before then switching to that same workspace if the delay of the mouse being there is done.
Basically, it would be nice to be able to have the window drag feature switch workspaces if the mouse is pushing or is near the place on the screen where the target workspace is. I know there's already a similar feature where you hold a window and use the hotkeys, but sometimes it can be quite hard to use the hotkey while still holding on the mouse button too.

**Comment by Vahan86** [comment]
Issue https://github.com/linuxmint/Cinnamon/issues/4889

**Comment by JosephMcc** [comment]
Actually it's this one https://github.com/linuxmint/Cinnamon/issues/5088
:)

**Comment by feren** [comment]
@JosephMcc Oh ok.
